fusion/model.py

```diff
@@ -0,0 +1,189 @@
+"""Model for fusion class"""
+import json
+import re
+from pydantic import BaseModel, validator, ValidationError, StrictStr, \
+    StrictInt, StrictBool
+from typing import Optional, List, Union
+
+
+class GeneDescriptor(BaseModel):
+    """Define a Gene, denoted by GeneDescriptor"""
+
+    type = 'GeneDescriptor'
+    value_id: str
+    label: StrictStr
+
+    @validator('value_id')
+    def is_valid(cls, v):
+        """Validate value_id"""
+        assert v.count(':') == 1 and v.find(' ') == -1, 'id must be a CURIE'
+        return v
+
+
+class SequenceLocation(BaseModel):
+    """Define SequenceLocation class"""
+
+    type = 'SequenceLocation'
+    seq_id: str
```

Comment:
Not a huge deal since this is practically the same, but wondering if we should stick with VRS naming conventions? 
```suggestion
    sequence_id: str
```